src/test/java/uk/gov/ons/census/casesvc/messaging/FulfilmentRequestReceiverIT.java

```diff
@@ -90,6 +100,70 @@ public void testFulfilmentRequestLogged() throws InterruptedException {
     FulfilmentRequestDTO actualFulfilmentRequest =
         convertJsonToFulfilmentRequestDTO(event.getEventPayload());
     assertThat(actualFulfilmentRequest.getCaseId()).isEqualTo(TEST_CASE_ID.toString());
-    assertThat(actualFulfilmentRequest.getFulfilmentCode()).isEqualTo(TEST_FULFILMENT_CODE);
+    assertThat(actualFulfilmentRequest.getFulfilmentCode())
+        .isEqualTo(TEST_REPLACEMENT_FULFILMENT_CODE);
+  }
+
+  @Test
+  public void testIndividualResponseFulfilmentRequestLogged()
+      throws InterruptedException, IOException {
+    // GIVEN
+    EasyRandom easyRandom = new EasyRandom();
+    Case caze = easyRandom.nextObject(Case.class);
+    caze.setCaseId(TEST_CASE_ID);
+    caze.setUacQidLinks(null);
+    caze.setEvents(null);
+    Case parentCase = caseRepository.saveAndFlush(caze);
+
+    ResponseManagementEvent managementEvent = getTestResponseManagementFulfilmentRequestedEvent();
+    managementEvent.getPayload().getFulfilmentRequest().setCaseId(TEST_CASE_ID.toString());
+    managementEvent
+        .getPayload()
+        .getFulfilmentRequest()
+        .setFulfilmentCode(TEST_INDIVIDUAL_RESPONSE_FULFILMENT_CODE);
+    managementEvent.getEvent().setTransactionId(UUID.randomUUID());
+
+    // WHEN
+    String json = convertObjectToJson(managementEvent);
+    Message message =
+        MessageBuilder.withBody(json.getBytes())
+            .setContentType(MessageProperties.CONTENT_TYPE_JSON)
+            .build();
+    rabbitQueueHelper.sendMessage(inboundQueue, message);
+
+    // wait for the the emitted event
+    BlockingQueue<String> outboundQueue = rabbitQueueHelper.listen(rhCaseQueue);
+    rabbitQueueHelper.checkExpectedMessageReceived(outboundQueue);
```

Comment:
This checks that a msg was sent, it would also be good to check the msg a little: 

copied from SampleReceiverIT:

ResponseManagementEvent responseManagementEvent =
        rabbitQueueHelper.checkExpectedMessageReceived(queue);
    assertEquals(EventType.CASE_CREATED, responseManagementEvent.getEvent().getType());

Maybe also test the EstabArid is there.  responseManagementEvent.payload.collectionCase.address.estabArid . == parent.estabArid

Also wouldn't hurt to stash this as an emittedCase and then compare Id to the saved one later, I know they'll match, but potentially nice